src/utils/react-admin/helper.js

```diff
@@ -0,0 +1,10 @@
+export const toSlug = value => value
+  .toLocaleLowerCase()
+  .normalize('NFD')
```

Comment:
![image](https://user-images.githubusercontent.com/1846633/67088279-08bcce80-f1a5-11e9-8940-44b167d49f3d.png)

Isn't this an issue? (ping @jrmi)